babbage/query/cuts.py

```diff
@@ -27,5 +27,5 @@ def apply(self, q, cuts):
             info.append({'ref': ref, 'operator': operator, 'value': value})
             table, column = self.cube.model[ref].bind(self.cube)
             q = self.ensure_table(q, table)
-            q = q.where(column == cast(value, column.type))
```

Comment:
removing cast while adding IN. Don't feel like dealing with casting each value and we're removing cast anyway.
We the result of this is a server error with SQL error message when the wrong like 

> "status": "error", "message": "(psycopg2.ProgrammingError) operator does not exist: text = integer\nLINE 4: WHERE incexp_labeled.item_code IN (100)) AS anon_1\n                                       ^\nHINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.\n"